tezos/context/src/working_tree/storage.rs

```diff
@@ -328,9 +335,15 @@ impl BlobId {
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct DirEntryId(u32);
 
-#[derive(Debug)]
+#[derive(Debug, Error)]
 pub struct DirEntryIdError;
 
+impl std::fmt::Display for DirEntryIdError {
```

Comment:
I think you can just add `#[error("Message")]` to the whole struct and it should work, example: https://github.com/dtolnay/thiserror/blob/master/tests/test_from.rs#L6-L11

Reply:
@tizoc Thanks